rosdep/python.yaml

```
@@ -6493,6 +6493,12 @@ python3-hypothesis:
   fedora: [python3-hypothesis]
   gentoo: [dev-python/hypothesis]
   ubuntu: [python3-hypothesis]
+python3-icecream:
+  debian:
+    bookworm: [python3-icecream]
+  ubuntu:
+    jammy: [python3-icecream]
+    noble: [python3-icecream]
```

We should use the future-proof syntax here so that we don't need to add future Ubuntu distributions manually.

```suggestion
  ubuntu:
    '*': [python3-icecream]
    focal: null
```